assets/js/atomic/blocks/product-elements/rating/edit.tsx

```diff
@@ -3,22 +3,40 @@
  */
 import { __ } from '@wordpress/i18n';
 import { useBlockProps } from '@wordpress/block-editor';
+import { BlockEditProps } from '@wordpress/blocks';
+import { useEffect } from 'react';
 
 /**
  * Internal dependencies
  */
 import Block from './block';
 import withProductSelector from '../shared/with-product-selector';
 import { BLOCK_TITLE, BLOCK_ICON } from './constants';
+import { Attributes } from './types';
+import { ProductQueryContext as Context } from '../../../../blocks/product-query/types';
```

Comment:
Just noting that I know this is not an ideal import, using the relative path. However, moving this to an alias includes updating/cleaning up the tsconfig(s) and webpack config - it would also be good to test any potential impact that might raise.

In the interest of keeping this PR scoped to supporting the PQ block, I propose addressing this in a follow-up item.

Reply:
What potential impacts do you see here? I agree with you on keeping the PR scoped, but also to not unnecessarily add any other small tasks which could be solved with little effort.

I see that perhaps you mean that if we introduce a `@woocommerce/blocks` alias path and migrate all our paths it could cause some unintended consequences, is that correct? While I'm not seeing too many unintended consequences for that either, we can add the alias and use it only here (and from now on) to keep this PR correctly scoped.

It's, in a sense, the same principle as not necessarily removing all our TS errors, but not add any further TS error. So here, we don't have to fix all relative paths, just not add wood to the fire, so to speak. And then, like you say, make another refactoring issue and PR to fix the older relative paths (or fix them as they come into play).

So, these are my 2c. But I'm approving the PR because it's such a small thing and if you disagree with it, I'm also ok with that.